src/Windows/Host/Client/Impl/BrokerServices/RemotingWebServer.cs

```diff
@@ -11,5 +13,13 @@ public class RemotingWebServer : IRemotingWebServer {
         public Task<string> CreateWebServerAsync(string remoteUrl, string baseAddress, string name, IActionLog log, IConsole console, CancellationToken ct = default(CancellationToken)) {
             return WebServer.CreateWebServerAsync(remoteUrl, baseAddress, name, log, console, ct);
         }
+
+        public Task<string> CreateLocalStaticFileServerAsync(string url, IFileSystem fs, IActionLog log, IConsole console, CancellationToken ct = default(CancellationToken)) {
```

Comment:
These also need better names, I think (currently they imply that they are factory methods for servers, but they don't return a server instance...).